docs/getting-started/quickstart.md

```diff
@@ -0,0 +1,143 @@
+# JDA (Java Discord API)
+
+This page provides a fast introduction to JDA.  For a more comprehensive guide, please view [the setup guide](../using-jda/getting-started.md). 
+
+## Download
+Whilst downloads are available on the [Jenkins Server](https://ci.dv8tion.net/job/JDA5/) and the [JDA GitHub releases](https://github.com/DV8FromTheWorld/JDA/releases) pages, 
+it is highly recommended to use a build tool like Gradle or Maven to manage JDA and its dependencies.
+
+!!! Information ""
+    === "Maven"
+        ```xml
+        <dependency>
+          <groupId>net.dv8tion</groupId>
+          <artifactId>JDA</artifactId>
+          <version>VERSION</version>
+        </dependency>
+        ```
+    === "Maven (No Audio)"
+        ```xml
+        <dependency>
+          <groupId>net.dv8tion</groupId>
+          <artifactId>JDA</artifactId>
+          <version>VERSION</version>
+          <exclusions>
+            <exclusion>
+              <groupId>club.minnced</groupId>
+              <artifactId>opus-java</artifactId>
+            </exclusion>
+          </exclusions>
+        </dependency>
+        ```
+    === "Gradle"
+        ```groovy
+        repositories {
+            mavenCentral()
+        }
+        
+        dependencies {
+            implementation("net.dv8tion:JDA:VERSION")
+        }
+        ```
+    === "Gradle (No Audio)"
+        ```groovy
+        repositories {
+            mavenCentral()
+        }
+        
+        dependencies {
+            implementation("net.dv8tion:JDA:VERSION") {
+            exclude module: 'opus-java'
+          }
+        }
+        ```
+    Remember to replace the `VERSION` with the version you would like to use.  Version information can be found from 
+    [JDA Discord Server](https://discord.gg/0hMr4ce0tIk3pSjp), [JDA GitHub](https://github.com/DV8FromTheWorld/JDA/releases) or 
+    [Maven Central](https://mvnrepository.com/artifact/net.dv8tion/JDA/).
+    
+    [![Download](https://img.shields.io/maven-central/v/net.dv8tion/JDA?color=blue)](https://ci.dv8tion.net/job/JDA5/lastSuccessfulBuild/)
```

Comment:
Why not a link to maven central, pointing to latest ? The CI server *could* be going down at some point